docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+# Backlog Structure for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
+
+## Roles
+* **Triager**: Person who is triaging the issue by determining its workability. This person is responsible for getting the ticket to one of two stages - 1) ready-for-work 2) wontfix. They are responsible for triaging  by working with the OP to get additional information as needed and analyzing the issue and adding relevant details/information/guidance that would be helpful to the resolution of the issue.
+
+* **OP**: Original Poster. This is the person who has opened or posted the issue.
+
+* **Contributor**: Person(s) that are actually doing the work related to the ticket. Once work is done, they work with the Reviewer and the Contributing Reviewers to get feedback implemented and complete the work. They are responsible for making sure issue status label is up to date.
+
+* **Reviewer**: Person(s) whose Approval is needed to merge the PR.
+
+* **Contributing Reviewer**: Person who provides feedback on an PR but whose Approval is not necessarily needed to merge the PR.
+
+## SLAs
+We value having a responsive backlog, and have response SLAs for each role. If an issue exceeds an SLA, it is put into the `stale` state.
+
+* **Triager**: Daily (Business Days) review of new issues with initial feedback to OP or issue processed within 3 days of last activity.
+* **Reviewer / Contributing Reviewer**: Provide feedback within 3 days of the last activity on a PR
+* **Contributor**: 3 day updates as comments on the PR to indicate activity
+
+## Issue States
+
+### New
+label: `new`
+
+New issues are the domain of the backlog Triagers. Every new issue should be responded to quickly, with a greeting and any needed clarifying questions. If assessing the issue requires more information, the Triager collaborates with the OP until workability is determined.
+
+The most common next steps for new tickets:
+* `available` for actionable work items.
+* `needs-discussion` for issues which cannot be triaged quickly, but raise important points.
+* `no-further-action` for issues which can be quickly resolved.
+
+### Available
+label: `available`
+
+For new work requests Once it has been classified, a ticket is up for grabs to be picked up to be worked on by a contributor.
```

Comment:
Typo: Once

Reply:
"Once a new work request has been classified any contributor may begin work on it" makes more sense than 
"a ticket is up for grabs to be picked up to be worked on by a contributor" if we're not using `up-for-grabs`.